python/ray/dashboard/client/src/pages/dashboard/logical-view/Actor.tsx

```diff
@@ -149,34 +139,46 @@ class Actor extends React.Component<Props & WithStyles<typeof styles>, State> {
                   .join(", "),
             },
             {
-              label: "Pending",
+              label: "Number of pending tasks",
               value: actor.taskQueueLength.toLocaleString(),
+              tooltip:
+                "The number of tasks that are currently pending to execute on this actor. If this number " +
+                "remains consistently high, it can indicate that this actor is a bottle-neck in your program.",
             },
             {
-              label: "Executed",
+              label: "Number of executed tasks",
               value: actor.numExecutedTasks.toLocaleString(),
+              tooltip:
+                "The number of tasks this actor has executed throughout its lifetimes.",
             },
             {
-              label: "NumObjectIdsInScope",
+              label: "Number of object IDs in scope",
               value: actor.numObjectIdsInScope.toLocaleString(),
+              tooltip:
+                "The number of objects ids that this actor is keeping in scope via its internal state. " +
+                "This does not imply that the objects are in active use or colocated on the node with the actor " +
+                "currently. This can be useful for debugging memory leaks.",
```

Comment:
```suggestion
                "The number of ObjectIDs that this actor currently has in scope. The objects for these IDs may exist on this node or a different one. If this number grows or doesn't decrease as expected, it may indicate a memory leak. For more information, see <link to docs>.",
```